src/Test/Spec/Reporter/Codewars.purs

```diff
@@ -1,51 +1,48 @@
-module Test.Spec.Reporter.Codewars
-       ( codewarsReporter
-       ) where
+module Test.Spec.Reporter.Codewars (codewarsReporter) where
 
 import Prelude
 
 import Data.Maybe (Maybe(..))
-import Data.String
-  ( Pattern(Pattern)
-  , Replacement(Replacement)
-  , replaceAll
-  )
-import Effect.Console (log)
-
+import Data.String (Pattern(..), Replacement(..), replaceAll)
+import Data.Time.Duration (Milliseconds(..))
+import Effect.Exception (message, stack)
+import Test.Spec.Console (tellLn, tellLns)
 import Test.Spec.Reporter.Base (defaultReporter)
+import Test.Spec.Result as Result
 import Test.Spec.Runner (Reporter)
 import Test.Spec.Runner.Event as Event
 
-
 codewarsReporter :: Reporter
 codewarsReporter = defaultReporter {} update
   where
-    update s = case _ of
-      Event.Start _ -> pure s
-      Event.End _ -> pure s
-      Event.Suite name -> s <$ log ("\n<DESCRIBE::>" <> name)
-      Event.SuiteEnd -> s <$ log "\n<COMPLETEDIN::>"
-
-      Event.Pending name -> s <$ do
-        log $ "\n<IT::>" <> name
-        log $ "\n<LOG::>Pending Test"
-        log $ "\n<COMPLETEDIN::>"
-
-      Event.Pass name _ ms -> s <$ do
-        log $ "\n<IT::>" <> name
-        log $ "\n<PASSED::>Test Passed"
-        log $ "\n<COMPLETEDIN::>" <> (show ms)
-
-      Event.Fail name msg mStack -> s <$ do
-        log $ "\n<IT::>" <> name
-        log $ "\n<FAILED::>Test Failed<:LF:>" <> (escapeLF msg)
-        case mStack of
-          Nothing -> pure unit
-          Just k  -> log $ "\n<LOG::-Stack Trace>" <> (escapeLF k)
-        log $ "\n<COMPLETEDIN::>"
-
-      _ -> pure s
-
+  update = case _ of
+    Event.Start _ -> pure unit
+    Event.End _ -> pure unit
+    Event.Suite _ex _path name -> tellLn ("<DESCRIBE::>" <> name)
```

Comment:
:information_source: `tellLn` will append (not prepend) a line break. This is different from before, but seems to be idiomatic. Please double check if this is working as expected. Otherwise we need to change it.

Reply:
The newline must be prepended to avoid outputs from user's solution from interfering. There should be a newline after it as well, so

```purescript
tellLn ("\n<DESCRIBE::>" <> name)
```
?

Reply:
Added a helper, will result in newline before and after: e40d7b623fd52b336fb3007b9fefa8644ed45a55

Output now looks like this: https://github.com/codewars/purescript-spec-reporter-codewars/pull/1/files#r965800824